### PR TITLE
[13.0][ADD] sale_product_variant_attribute_tax: New module

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT27131

Depends on:
- [x] https://github.com/OCA/product-variant/pull/205

### CODE REVIEW

**Review by joao-p-marques — COMMENTED** [review]
Code review :+1: 

Please check Travis. Seems like a dependency that needs to be added in `oca_dependencies.txt`

File: sale_product_variant_attribute_tax/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    @api.onchange("product_attribute_ids")
+    def _onchange_product_attribute_ids_configurator(self):
+        res = super()._onchange_product_attribute_ids_configurator()
+        # if there's a product set, taxes on that product have priority
+        if self.product_id or not self.product_tmpl_id:
+            return res
+        values = self.mapped("product_attribute_ids.value_id")
+        taxes = values.mapped("tax_ids").filtered(lambda x: x.type_tax_use == "sale")
+        # Filter taxes not belonging to the current company
+        taxes = taxes.filtered(lambda x: x.company_id == self.order_id.company_id)
```
Comment (pedrobaeza) [inline sale_product_variant_attribute_tax/models/sale_order.py]: As you are doing a filtered on the previous line, do it at once both.

**Comment by ernestotejeda** [comment]
@pedrobaeza changes done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**